src/error.ts

```diff
@@ -80,6 +80,17 @@ export const GET_MORE_RESUMABLE_CODES = new Set<number>([
   MONGODB_ERROR_CODES.CursorNotFound
 ]);
 
+/** @public */
+export const MONGODB_ERROR_LABELS = Object.freeze({
+  RetryableWriteError: 'RetryableWriteError',
+  TransientTransactionError: 'TransientTransactionError',
+  UnknownTransactionCommitResult: 'UnknownTransactionCommitResult',
+  ResumableChangeStreamError: 'ResumableChangeStreamError'
+} as const);
+
+/** @public */
+export type MONGODB_ERROR_LABELS = typeof MONGODB_ERROR_LABELS[keyof typeof MONGODB_ERROR_LABELS];
```

Comment:
It might be confusing to read but it does give the expected behavior of an enum, which is a container for a set of values, and also enumerates the possible values. I do thing the all caps style doesn't fit with our other enums, udpated!